.binder/postBuild

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+
+# Taken from https://github.com/scikit-learn/scikit-learn/blob/72b3041ed57e42817e4c5c9853b3a2597cab3654/.binder/postBuild
+# under BSD3 license, copyright the scikit-learn contributors
+
+python -m pip install .
+
+# This script is called in a binder context. When this script is called, we are
+# inside a git checkout of the scikit-image/scikit-image repo. This script
+# generates notebooks from the scikit-image python examples.
+
+if [[ ! -f /.dockerenv ]]; then
+    echo "This script was written for repo2docker and is supposed to run inside a docker container."
+    echo "Exiting because this script can delete data if run outside of a docker container."
+    exit 1
+fi
+
+# Copy content we need from the scikit-image repo
+TMP_CONTENT_DIR=/tmp/scikit-image
+mkdir -p $TMP_CONTENT_DIR
+cp -r doc/examples .binder $TMP_CONTENT_DIR
+# delete everything in current directory including dot files and dot folders
+# to create a "clean" experience for readers
+find . -delete
+
+# Generate notebooks and remove other files from examples folder
+GENERATED_NOTEBOOKS_DIR=notebooks
+cp -r $TMP_CONTENT_DIR/examples $GENERATED_NOTEBOOKS_DIR
+
+find $GENERATED_NOTEBOOKS_DIR -name '*.py' -exec sphx_glr_python_to_jupyter.py '{}' +
```

Comment:
`sphx_glr_python_to_jupyter.py` is somewhere in the path thanks to `sphinx-gallery`, right?

Reply:
Nods.